src/Home_excercise_5.java

```diff
@@ -0,0 +1,116 @@
+import java.util.Locale;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Home_excercise_5 {
+
+        public static void main(String[] args) {
+            Scanner scanner = new Scanner(System.in);
+            Scanner sc1 = new Scanner(System.in);
+            System.out.print("Enter your text: ");
+            String yourString = scanner.nextLine();
+            System.out.print("Enter your symbol: ");
+            String yourCharacter = sc1.nextLine();
+            int amountSymbols = findSymbolOccurance1(yourString,yourCharacter);
+            System.out.println("Total same symbols: "+amountSymbols);
+            // int countSymbol = findSymbolOccurance2(yourString,yourCharacter);//It's my the second version, but
+            //something wrong, please HELP!!!
+            // System.out.println("Amount of symbol: "+countSymbol);
+
+            Scanner sc2 = new Scanner(System.in);
+            System.out.print("Enter your next text: ");
+            String nextText = sc2.nextLine();
+            Scanner sc3 = new Scanner(System.in);
+            System.out.print("Enter your check text: ");
+            String checkText = sc3.nextLine();
+            int indexFirstSymbol = findWordPosition(nextText,checkText);
+            System.out.println("The index of the first symbol: "+indexFirstSymbol);
+            System.out.println("The revers your text: "+stringReverse(nextText));
+            System.out.println("Your text is Palindrome: " + isPalindrome(nextText));
+
+            Random random = new Random();
+            String [] secretWord = {"apple", "orange", "lemon", "banana", "apricot", "avocado", "broccoli", "carrot",
+                    "cherry", "garlic", "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", " pea",
+                    "peanut", "pear", "pepper", "pineapple", "pumpkin", "potato"};
+            Scanner sc4 = new Scanner(System.in);
+
+            int indexElement = random.nextInt(secretWord.length);
+            String randomWord = secretWord[indexElement];
+
+            //System.out.println(indexElement);
```

Comment:
Please do not leave commented lines, it's a rule of thumb that they should be removed